database/rrdhost.c

```diff
@@ -534,6 +537,13 @@ void rrdhost_free(RRDHOST *host) {
     while(host->alarms)
         rrdcalc_unlink_and_free(host, host->alarms);
 
+    while(host->health_rep_alarm_entry_list) {
```

Comment:
I would put this code inside health_alarm_log_free. It seems to me a bit cleaner that way...

Reply:
OK, I'll do that.

Reply:
These codes are not longer part of the change.